lib/master.js

```diff
@@ -24,10 +24,10 @@ var default_options = {
 	connect_timeout:     3  * 1000, // 3s
 	ping_timeout:        10 * 1000, // 10s
 	monitor_interval:    1  * 1000, // 1s
-	totalmaxheap:        512 * 1024 * 1024, // 512 MB
+	total_maxheap_mb:    512, // 512 MB
```

Comment:
I see what you're saying (your comment has helped me take off a blindfold!). One of the reason I was doing this change was to allow our config to be passed directly from static config file down to ipcluster, and I didn't want to have ops have to change the configs (currently specified in MB). However This is our app's consideration. ipcluster shouldn't have to care about this and bytes do indeed give the best granularity/control.

But... with all that said, the overhead of node itself means that the rss of any node process can not be less than 1MB.... So does setting thresholds in bytes makes sense, even from ipcluster's perspective?